### PR TITLE
Textual improvements on 'Connection pool management'

### PR DESCRIPTION
**Pull request by ali-ince**
Made a couple of textual improvements in `Connection pool management` section.

### CODE REVIEW

File: src/main/asciidoc/client-applications.adoc [hunk 1]
```diff
@@ -388,16 +388,16 @@ Detailed descriptions of connection pool settings available via driver configura
...
-It is recommended to set driver's maximum lifetime to a value smaller than the maximum lifetime configured in its application system infrastructure (such as operation system, router, load balancer, proxy and firewall).
+It is recommended to pick a value smaller than the maximum lifetime exposed by the surrounding system infrastructure (such as operating system, router, load balancer, proxy and firewall).
 Negative values result in lifetime not being checked.
 Default value: 1h.
 
 `MaxConnectionPoolSize`::
-This setting defines the maximum total number of connections allowed to be managed by the connection pool on each host.
-In other words, for a direct driver, this sets the maximum amount of connections towards a single database.
-For a routing driver this sets the maximum amount of connections towards each cluster member.
+This setting defines the maximum total number of connections allowed to be managed by the connection pool per host.
```
Comment (mariascharin) [inline src/main/asciidoc/client-applications.adoc]: Proposed wording to make the sentence more clear:
This setting defines the maximum total number of connections allowed, per host, to be managed by the connection pool.
Please verify that the meaning is still correct. Thanks!

**Review by mariascharin — APPROVED** [review]
@ali-ince 
Good changes! Thank you! 
I made a minor comment.

I noted that your local branch is called _1.6-xxx_ but that you've opened the PR against 1.5. I think the change should indeed be done in 1.5, but wanted to make sure that's your intention too.

**Comment by ali-ince** [comment]
Thanks for the comments @mariascharin. I've applied your proposed wording and since the changes are not specific to a version I think it's ok that we merge this into `1.5` and forward merge to `1.6`.
